### PR TITLE
feat: [sc-107] Calendar View: Navigate between months

### PR DESCRIPTION
**Pull request by mjrj97**
Story details: https://app.shortcut.com/5-semester/story/107

Not a big fan of the gray, but was just a proof of concept.

image

### CODE REVIEW

**Comment by gubbih** [comment]
i get that, i thinks its because its not filled, this is sling, a calender/shift work sheet
image

File: src/calendar/components/CalendarCell.tsx [hunk 1]
```diff
@@ -36,7 +38,12 @@ export default function CalendarCell({
 
   return (
     <td
-      style={{ verticalAlign: 'top', borderTop: '1px solid', height: 64 }}
+      style={{
```
Comment (gubbih) [inline src/calendar/components/CalendarCell.tsx:41]: Is there a reason for not using a stylesheet file?

**Review by gubbih — DISMISSED** [review]

File: src/calendar/tests/Event.fixtures.ts [hunk 2]
```diff
@@ -36,6 +36,9 @@ const events: Event[] = [
   },
 ];
 
-const eventsAtom = atom<Event[] | undefined>({ key: 'mockedEvents', default: events });
+const eventsAtom = atom<Event[] | undefined>({
```
[the remark this answers is not in the document]
Comment (mjrj97) [reply src/calendar/tests/Event.fixtures.ts:39]: Fixed in https://github.com/5-semesterprojekt/Frontend/pull/19

File: src/calendar/components/CalendarCell.tsx (hunk 1 above)
Comment (mjrj97) [reply src/calendar/components/CalendarCell.tsx:41]: What do you mean?

File: src/calendar/tests/Event.fixtures.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (gubbih) [reply src/calendar/tests/Event.fixtures.ts:39]: image

File: src/calendar/components/CalendarCell.tsx (hunk 1 above)
Comment (gubbih) [reply src/calendar/components/CalendarCell.tsx:41]: like a CSS file, with styling.

Comment (mjrj97) [reply src/calendar/components/CalendarCell.tsx:41]: oh lol read it as 'spreadsheet file'

I usually use inline styles whenever I don't need to reuse a style, but sure if I need to reuse a css class, I'll make a css file.

Comment (gubbih) [reply src/calendar/components/CalendarCell.tsx:41]: just a thought

**Pull request merged**